src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "utils/variables.scss";
+
+@import "blocks/calendar.scss";
+@import "blocks/start-day.scss";
+@import "blocks/month-length.scss";
+
 body {
+  display: flex;
+  justify-content: center;
+  text-align: center;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
```

Comment:
```suggestion

```
Consider moving these styles to a separate file for defaults to keep your main file clean and clear